web/src/components/tasks/business-formation/contacts/Addresses.tsx

```diff
@@ -179,80 +196,104 @@ export const Addresses = <T extends FormationMember | FormationIncorporator>(
     </table>
   );
 
-  const renderMobileTable =
-    props.addressData.length > 0 ? (
-      <>
-        <table
-          data-testid={`addresses-${props.fieldName}-table-mobile`}
-          className={`addresses-mobile margin-y-2`}
-        >
-          <tbody>
-            {props.addressData.map((it, index) => {
+  const renderMobileTable = (
+    <>
```

Comment:
[dust] I see we deleted the conditional rendering on this component when addressData.length > 0

Was there a change in design making it so we want this to appear empty sometimes?

Reply:
The layouts didn't match the Figma designs.

Reply:
The layout in mobile view didn't match Figma.